morphocut/str.py

```diff
@@ -36,6 +36,8 @@ class Format(Node):
 
             with Pipeline() as pipeline:
                 result = Format(fmt, *args, _args=_args, _kwargs=_kwargs, **kwargs)
+    
+    The produces the output: "1,2,3,4,5,6,7,8,9,10"
```

Comment:
Indent by 4 spaces.

`` Result: `obj[result]` == `"1,2,3,4,5,6,7,8,9,10"` for a stream object `obj`.``